src/ERC1155A.sol

```diff
@@ -509,24 +502,32 @@ abstract contract ERC1155A is IERC1155A {
             }
         }
 
-        emit TransferBatch(msg.sender, address(0), to, ids, amounts);
+        emit TransferBatch(operator, address(0), to, ids, amounts);
```

Comment:
string based error message above, line 492 (could be custom error)

Reply:
Fixed